lib/node.js

```diff
@@ -822,7 +822,7 @@ declare module "http" {
     listen(port: number, hostname?: string, backlog?: number, callback?: Function): Server;
     listen(path: string, callback?: Function): Server;
     listen(handle: Object, callback?: Function): Server;
-    close(callback?: Function): Server;
+    close(callback?: (error: ?Error) => void): Server;
```

Comment:
Nowadays we tend to use `mixed` as the return type for callbacks when the callback return value is ignored. I'll update this before logging!